<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalzemeGiris</class>
 <widget class="QWidget" name="MalzemeGiris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMusteriAdi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Müşteri Adı</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMalzemeAdi">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Malzeme Adı</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMalzemeAdi">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMusteriAdi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblMiktar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Miktar(Br)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBirimFiyat">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Birim Fiyat (YTL)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFire">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fire (%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblToplam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Toplam = 1 YTL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBorc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>721</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mevcut Borç = 10 YTL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblKalan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>691</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kalan = 11 YTL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Net(Br)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMiktar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFire">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNet">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBirimFiyat">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKaydet">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOdenen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblOdenen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödenen (YTL)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFireKg">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblFireKg">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fire(Br)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZaman">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
